src/commands/deployWorkspaceProject/deployWorkspaceProject.ts

```diff
@@ -25,25 +28,27 @@ export async function deployWorkspaceProject(context: IActionContext & Partial<D
         item = undefined;
     }
 
-    const subscription: AzureSubscription = await subscriptionExperience(context, ext.rgApiV2.resources.azureResourceTreeDataProvider);
+    const subscription: AzureSubscription = item?.subscription ?? await subscriptionExperience(context, ext.rgApiV2.resources.azureResourceTreeDataProvider);
     const subscriptionContext: ISubscriptionContext = createSubscriptionContext(subscription);
-
-    // Show loading indicator while we configure default values
-    let defaultContextValues: Partial<DeployWorkspaceProjectContext> | undefined;
-    await window.withProgress({
-        location: ProgressLocation.Notification,
-        cancellable: false,
-        title: localize('loadingWorkspaceTitle', 'Loading workspace project deployment configurations...')
-    }, async () => {
-        defaultContextValues = await getDefaultContextValues({ ...context, ...subscriptionContext }, item);
-    });
-
-    const deployWorkspaceProjectInternalContext: DeployWorkspaceProjectInternalContext = Object.assign(context, {
-        ...defaultContextValues,
+    const containerAppContext: IContainerAppContext = Object.assign(context, {
         ...subscriptionContext,
         subscription
     });
 
+    let deploymentConfiguration: DeploymentConfiguration;
```

Comment:
All I did here was rename `deployWorkspaceProjectV2` to `deployWorkspaceProject`.  Since the old file shares this same name, it did a diff on this file, but it's really just a copy of the old v2 logic.